CSharp/Library/Microsoft.Bot.Connector.Shared/StateAPI/Models/BotData.cs

```diff
@@ -21,9 +21,8 @@ public partial class BotData
         /// <summary>
         /// Initializes a new instance of the BotData class.
         /// </summary>
-        public BotData(string eTag = default(string), object data = default(object))
+        public BotData(string eTag = @"*", object data = default(object))
```

Comment:
apparently. @matvelloso and I were told this should be what devs do when playing with the UserData via the State Client in a discussion regarding this behavior during a hackfest a number of weeks ago. "always set etag to '*'"